packages/components/src/components/text-area/_text-area.scss

```diff
@@ -99,6 +99,14 @@
       color: transparent;
     }
   }
+
+  .#{$prefix}--text-area__counter-wrapper {
+    width: 100%;
+  }
+
+  .#{$prefix}--text-area__counter {
+    float: right;
```

Comment:
Would it be possible to use flex box to position the text area counter instead of floats?

Reply:
you wish has been granted